packages/core/src/fetchers/fetchReferencedEntities.ts

```diff
@@ -56,8 +56,8 @@ export const fetchReferencedEntities = async ({
   }
 
   const [entriesResponse, assetsResponse] = (await Promise.all([
-    fetchAllEntities({ client, entityType: 'Entry', ids: entryIds, locale }),
-    fetchAllEntities({ client, entityType: 'Asset', ids: assetIds, locale }),
+    fetchAllEntries({ client, ids: entryIds, locale }),
+    fetchAllAssets({ client, ids: assetIds, locale }),
   ])) as unknown as [MinimalEntryCollection, AssetCollection];
```

Comment:
It would be great to get rid of this `as unknown` cast. 

Now it can be considered leftover from the time when I put it because I was fetching directly via ` client.withoutLinkResolution.getEntries({ ...query });` and this was the only way I found  to wrangle types. 

Now that we have control over `fetchAllEntries()`  we unlikely need it (and this cast is kinda dangerous)